addons/project/models/project_task.py

```diff
@@ -1119,7 +1119,7 @@ def update_date_end(self, stage_id):
     # Subtasks
     # ---------------------------------------------------
 
-    @api.depends('parent_id.partner_id', 'project_id', 'is_private')
+    @api.depends('parent_id', 'project_id', 'is_private')
```

Comment:
Unrelated to your change. 🤔 

Bad rebase?

Reply:
Oops, I think I included this change from 16.2 by accident :sweat_smile: I changed it back and pushed. Should be good now